Environmental Sensing/Radon Sensor Signoff.md

```diff
@@ -0,0 +1,77 @@
+
+# Constraints:                                                                                                                                  
+
+1. The sensor must be able to read radon levels from 0.5pCi/L to 4.5pCi/L 
+    - The average level for a home is 1.3 pCi/L
+    - EPA recommends you should consider fixing radon levels if greater than 2pCi/L
+2. The sensor must be able to communicate to the microcontroller using a communicative language (I2C, UART, or SPI protocols)
+    - This will allow the sensor to use the same MCU as other sensors
+3. The sensor must have accuracy within .5pCi/L for good readings
+4. The sensor should be powered by either 5 or 12V.
+5. The sensor should have both short-term and long-term functionality.
+
+
+
+# Analysis
+
+| Type                     | First reliable data out | Data interval                       | Sensitivity                           | Operating range   | Range             | Precision        | Accuracy                        | Power                               | Size                     |
+|--------------------------|-------------------------|-------------------------------------|---------------------------------------|-------------------|-------------------|------------------|---------------------------------|-------------------------------------|--------------------------|
+| pulsed ion chamber 200cc | < 60min                 | 10min update (60min moving average) | 0.5cpm/pCi/l at 10pCi/l (30cph/pCi/l) | 10~40°C, RH < 90% | 0.1 ~ 99.99 pCi/l | < 10% at 10pCi/l | < ±10% (min. error <±0.5pCi/l ) | DC 12 ± 0.1V, 65mA (12V DC adapter) | Φ80(mm) x 120(mm) , 240g |
+
+Figure 1
+This data is directly pulled from the product’s datasheet
+
+## Constraint 1:
+
+The radon sensor is able to measure between 0.1 to 99.9 pCi/L. These values work extremely well for what is needed.
+
+## Constraint 2:
+
+The sensor has a UART interface that will be used to communicate with the Arduino Nano. This will allow for data to be easier to track. The electrical schematic will show how these connections will take place.
+
+![Capture](https://github.com/JoshuaEgwuatu/Fall-2023-Autonomous-Crawlspace-Inspection-Robot/assets/110966922/8c94b023-226e-47b6-a670-f79a3ce60f19)
+An example of how we will communicate with the sensor
+
+## Constraint 3:
+
+The sensor accuracy meets the accuracy requirement of 0.5pCi/L or less. The sensor has been proven to have at least 90% accuracy according to Radon FTLab.
+
+## Constraint 4:
+
+The sensor is powered by corded electricity with a voltage input of 12 volts and an amperage of 0.1 amps. The group has decided to use another battery pack to power this sensor separately from the rest of the sensors to not take away from the overall battery life of the robot.
+
+## Constraint 5:
```

Comment:
I'm not certain what the constraint means. And what does it mean to display in intervals of 10 mins? How long does it take to get a measurement? How fast should the measurements be obtainable?